devices/webcam/hand_tracking_utility.py

```diff
@@ -0,0 +1,600 @@
+#!/usr/bin/env python3
```

Comment:
This file is relatively large and difficult to be reused by other applications, consider breaking it down into smaller files.

Reply:
Should be resolved in the latest commit